.ddev/commands/cypress/cypress-run

```diff
@@ -2,6 +2,19 @@
 ## Description: Use Cypress in "runner" mode.
 ##
 ## Usage: cypress-run [arguments]
-## Example: "ddev cypress-run --browser chrome"
+## Example: "ddev cypress-run"
+## This executes only the functional test cases using Electron.
 
-DISPLAY= cypress run "$@"
+# If this is run from the command line, these variables may not be set.
+# Set them to defaults. It may be required to reset the user password
+# using drush, like so:
+# `ddev drush upwd ct-admin ct-admin`
+# Or simply set the actual password in the environment variable.
+if [ -z "${CYPRESS_ADMIN_USERNAME}" ]; then
+  CYPRESS_ADMIN_USERNAME=ct-admin
+fi
+if [ -z "${CYPRESS_ADMIN_PASSWORD}" ]; then
+  CYPRESS_ADMIN_PASSWORD=ct-admin
+fi
```

Comment:
@zeshanziya, is there a better way to accomplish this?

Reply:
How much of a challenge is it to run `drush` reliably from within the Cypress container?

Reply:
Otherwise, setting the default username/password in docker-compose sounds good. We can leave a comment in `cypress-run` as that may be the first place people look when things don't work.

Reply:
> How much of a challenge is it to run `drush` reliably from within the Cypress container?

I believe it would be challenging, as the image is based on cypress/included, and PHP is not installed there. We may have to bring many dependencies from the web container to the Cypress container.